js/multi.js

```diff
@@ -0,0 +1,440 @@
+            'takerOrMaker': undefined,
+            'type': undefined,
+            'fee': undefined,
+        };
+    }
+
+    async fetchTicker (symbol, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const marketId = market['id'];
+        const response = await this.publicGetMarketStatusAll (params);
+        const marketTicket = this.getMarketTicket (response, marketId);
+        return this.parseTicker (marketTicket['result'], symbol);
+    }
+
+    getMarketTicket (response, marketId) {
+        const marketTicker = {};
+        for (let i = 0; i < response.length; i++) {
+            if (response[i]['market'] === marketId) {
+                marketTicker['result'] = response[i];
+                break;
+            }
+        }
+        return marketTicker;
+    }
+
+    parseTicker (ticker, symbol) {
+        return {
+            'symbol': symbol,
+            'info': ticker,
+            'high': ticker['high'],
+            'low': ticker['low'],
+            'bid': ticker['bid'],
+            'bidVolume': ticker['volume'],
+            'ask': ticker['ask'],
+            'open': ticker['open'],
+            'close': ticker['close'],
+            'last': ticker['close'],
+            'baseVolume': ticker['baseVolume'],
+            'quoteVolume': ticker['pairVolume'],
+            'askVolume': undefined,
+            'average': undefined,
+            'change': undefined,
+            'datetime': undefined,
+            'percentage': undefined,
+            'previousClose': undefined,
+            'timestamp': undefined,
+            'vwap': undefined,
+        };
+    }
+
+    async fetchBalance (params = {}) {
+        await this.loadMarkets ();
+        const response = await this.privateGetAssetBalance (params);
+        const exchange = response['exchange'];
+        const keys = Object.keys (exchange);
+        const result = { 'info': exchange };
+        for (let i = 0; i < keys.length; i++) {
+            const code = keys[i];
+            result[code] = {
+                'free': parseFloat (exchange[code]['available']),
+                'used': parseFloat (exchange[code]['freeze']),
+            };
+        }
+        return this.parseBalance (result);
+    }
+
+    async fetchDepositAddress (code, params = {}) {
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        const request = {
+            'symbol': currency['code'],
+        };
+        const response = await this.privatePostAssetDeposit (this.extend (request, params));
+        const currencyObject = this.safeValue (response, code);
+        return {
+            'currency': code,
+            'address': currencyObject['address'],
+            'tag': this.safeValue (currencyObject, 'memo'),
+            'info': currencyObject,
+        };
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        await this.loadMarkets ();
```

Comment:
Need to check how to use stopLimit. https://github.com/ccxt/ccxt/wiki/Manual#other-order-types